lib/reporters/cli/index.js

```diff
@@ -288,6 +294,49 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
 
 _.assignIn(PostmanCLIReporter, {
 
+    verboseSession (run) {
+        var verboseSession = run.verboseSession;
+
+        return verboseSession;
+    },
+
+    parseSingleRequestStatistics (run) {
+        const summaryTable = new Table({
+                style: { head: [] },
+                colWidths: [40]
+            }),
+            timings = run.timings,
+            transfers = run.transfers,
+            timingPhases = run.timingPhases;
+            verboseSession = run.verboseSession;
+        // add row to show total data received
+        transfers &&
+        summaryTable.push([colors.green('Data received:'), util.filesize(transfers.responseTotal) + ' (approx)']);
+
+        // add rows containing average time of different request phases
+        timings && _.forEach({
+            response: 'Response time:',
+            dns: 'DNS lookup time:',
+            firstByte: 'First byte time:'
+        }, (label, key) => {
+            timings[`${key}Average`] &&
+            summaryTable.push([colors.green(label), util.prettyms(timings[`${key}Average`])]);
+        });
+
+        timingPhases && _.forEach({
+            prepare: 'Prepare:',
+            tcp: 'TCP:',
+            firstByte: 'FirstByte:',
+            download: 'Download:',
+            secureHandshake: 'SecureHandshake:'
+        }, (label, key) => {
+            timingPhases &&
+            summaryTable.push([colors.green(label), util.prettyms(timingPhases[`${key}`]) + 'ms']);
```

Comment:
`util.prettyms` is already appends the unit to the value. Currently it is printing like `56msms`